packages/back-end/src/controllers/login.ts

```diff
@@ -22,9 +19,12 @@ export default ({
 
     const { email, password } = req.body;
 
-    const userProperties = await databaseService
-      .getUserByEmail(email)
-      .then((user) => user?.properties);
+    const resUser = await databaseService.getUserByEmail(email);
+    if (resUser === undefined) {
+      res.status(500).json({ message: 'Database server error.' });
+      return;
+    }
+    const userProperties = resUser.properties;
```

Comment:
That's not a 500, as the database is allowed to return undefined if the email is not found. Not sure why the implementation changed here, was there a bug?